components/create-bounty-details.tsx

```diff
@@ -15,6 +16,17 @@ export default function CreateBountyDetails({
   review = false,
 }) {
   const { t } = useTranslation("bounty");
+  const [errorTitle, setErrorTitle] = useState<boolean>(false);
```

Comment:
there's no need to have `setErrorTitle` as we know the length of the title, so we can simply do `title.length > limit ? show-error : null `